backend/src/main/services/BusinessAccountService.ts

```diff
@@ -84,6 +86,12 @@ export class BusinessAccountService {
       businessCreationRequestDTO.socialMediaInfo.businessId = business.id;
       await this.socialMediaPageService.createSocialMediaPage(businessCreationRequestDTO.socialMediaInfo);
     }
+    // Temporary email template.
+    const emailContent = `<p>Hi ${businessCreationRequestDTO.account.firstName}, here is your credential</p>
```

Comment:
It should be "here are your credentials". Correct this mistake for all the email templates.